french/appelle-genie/appelle-genie.md

```diff
@@ -0,0 +1,3 @@
+Vous sortez votre lampe magique que vous aviez oublié dans votre poche et invoquez le génie de la lampe. Il vous rappelle que vous avez déja exaucer vos trois souhaits et retourne dans sa lampe vous laissant desesperé et rempli de desespoir.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.